common/traits/wc_race_traits.txt

```diff
@@ -0,0 +1,1488 @@
+				}
+				desc = trait_creature_watcher_desc
+			}
+			desc = trait_creature_watcher_character_desc
+		}
+	}
+}
+creature_wolvar = {
+	index = 2080
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_wolvar_desc
+			}
+			desc = trait_creature_wolvar_character_desc
+		}
+	}
+}
+creature_taunka = {
+	index = 2081
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_taunka_desc
+			}
+			desc = trait_creature_taunka_character_desc
+		}
+	}
+}
+creature_kvaldir = {
+	index = 2082
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_kvaldir_desc
+			}
+			desc = trait_creature_kvaldir_character_desc
+		}
+	}
+}
+creature_nerubian = {
+	index = 2083
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_nerubian_desc
+			}
+			desc = trait_creature_nerubian_character_desc
+		}
+	}
+}
+creature_harpy = {
+	index = 2084
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_harpy_desc
+			}
+			desc = trait_creature_harpy_character_desc
+		}
+	}
+}
+creature_mechagnome = {
+	index = 2085
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_mechagnome_desc
+			}
+			desc = trait_creature_mechagnome_character_desc
+		}
+	}
+}
+creature_earthen = {
+	index = 2086
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_earthen_desc
+			}
+			desc = trait_creature_earthen_character_desc
+		}
+	}
+}
+
+# Bronze dragons
+creature_bronze_whelp = {
+	index = 2087
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_bronze_whelp_desc
+			}
+			desc = trait_creature_bronze_whelp_character_desc
+		}
+	}
+}
+creature_bronze_drake = {
+	index = 2088
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_bronze_drake_desc
+			}
+			desc = trait_creature_bronze_drake_character_desc
+		}
+	}
+}
+creature_bronze_dragon = {
+	index = 2089
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_bronze_dragon_desc
+			}
+			desc = trait_creature_bronze_dragon_character_desc
+		}
+	}
+}
+creature_bronze_wyrm = {
+	index = 2090
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_bronze_wyrm_desc
+			}
+			desc = trait_creature_bronze_wyrm_character_desc
+		}
+	}
+}
+
+creature_giant = {
```

Comment:
ok but
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/98306360-baa78280-1fd4-11eb-9362-e3bc777c2615.png)

</details>

Reply:
Changed in 63715f5fc6034c4596228752c75ab4708b7b08d9.